File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):
    split = binomial.split(' ')
    a = int(split[0])
    b = int(split[2])
    x = split[1]
    
    if x == '+':
        return  a + b
    elif x == '-':
        return a - b
    return a * b

File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {
        String[] str = binomial.split(" ");
        if (str[1].equals("+")) {
            return Integer.parseInt(str[0]) + Integer.valueOf(str[2]);
        } else if (str[1].equals("-")) {
            return Integer.parseInt(str[0]) - Integer.valueOf(str[2]);
        }
        return Integer.parseInt(str[0]) * Integer.valueOf(str[2]);
    }
}